### PR TITLE
[@mantine-dates] RangeCalendar & DateRangePicker: multiple months

### LINKED ISSUE #396: [Feature Request] RangePicker - option to show two months
As the title says, add an option to the RangePicker component to show two months next to each other. See the example below.

image

If you're ok with this feature, I'm willing to implement this.

### PR DESCRIPTION
**Pull request by MiguelKorn**
Had some issues with the other pr (#404), bud did implement the feedback. This pr fixes #396

### CODE REVIEW

File: docs/src/docs/dates/date-range-picker.mdx [hunk 1]
```diff
@@ -18,10 +18,16 @@ import { DateRangePickerDemos } from '@demos/dates';
...
+
+You can show two months next to each other by setting `amountOfMonths`:
+
+<Demo data={DateRangePickerDemos.multiMonth} />
+
 ## DatePicker props
 
 DateRangePicker component uses the same props as [DatePicker](/dates/date-picker/) component.
-The only difference is in `value` and `onChange` props shown in previous example.
+The only difference is in `value`, `onChange` and `withMultipleMonths` props shown in previous examples.
```
Comment (rtivital) [inline docs/src/docs/dates/date-range-picker.mdx:30]: `withMultipleMonths`? Shouldn't it be `amountOfMonths`?

Comment (MiguelKorn) [reply docs/src/docs/dates/date-range-picker.mdx]: You're right. Changed it to `amountOfMonths`

**Comment by rtivital** [comment]
As I'm thinking about it – there is not any good reason to restrict multiple months to range components – it can be used with DatePicker and Calendar as well – this feature should be implemented on `CalendarWrapper` component. Do you want to work on this or you would want me to merge this and leave this work for maintainers?

**Comment by MiguelKorn** [comment]
I'd rather let this be merged and leave it for maintainers

**Comment by rtivital** [comment]
Okay, thanks for your work, we appreciate that!

**Pull request merged**